source/index-management/index-management.rst

```diff
@@ -1031,7 +1066,7 @@ Index View API for Search Indexes
     /**
      * Updates a single search index from the collection by the index name.
      */
-    updateOne(name: String, options: Optional<UpdateSearchIndexOptions>): Result;
+    updateOne(name: String, definition: Document, options: Optional<UpdateSearchIndexOptions>): Result;
```

Comment:
it feels inconsistent to have name sometimes as the first parameter and sometimes as the second when both `name` and `definition` are potential parameters; if definition is always required and name is sometimes optional, it would indicate that we want to use `definition` always as the first parameter

Reply:
I view the `create` APIs separate from the `list`/`update`/`drop` apis.  Whereas the create APIs are saying "create a search index with definition <definition> and optional name <name>", the other APIs (specifically `drop` and `update`) are saying "do something to the index with name <name>".  Specifically, in this case, we're updating index with name <name> to have the new definition - flipping the parameters makes less sense.